### PR TITLE
Add render tests for CanvasRenderingContext2D.reset()

### PR DESCRIPTION
**Pull request by tuankiet65**
Previous tests check that the state are reset back to the default value, but doesn't actually check that the render result is correct.

### CODE REVIEW

**Review by annevk — APPROVED** [review]
Nice! Can you land these or should I land them for you?

**Comment by tuankiet65** [comment]
I don't have write access to this repo so I'll need some help to land it. Once the checks are done can you land this for me? (I just pushed a new commit to fix a typo)

**Pull request merged**
